src/UglyToad.PdfPig/Geometry/PdfPoint.cs

```diff
@@ -77,6 +77,17 @@ public PdfPoint MoveY(decimal dy)
             return new PdfPoint(X, Y + dy);
         }
 
+        /// <summary>
+        /// Creates a new <see cref="PdfPoint"/> which is the current point moved in the x and y directions relative to its current position by a value.
+        /// </summary>
+        /// <param name="dx">The distance to move the point in the x direction relative to its current location.</param>
+        /// <param name="dy">The distance to move the point in the y direction relative to its current location.</param>
+        /// <returns>A new point shifted on the y axis by the given delta value.</returns>
+        public PdfPoint MoveXY(decimal dx, decimal dy)
```

Comment:
It might be worth naming this `Translate` which is the name used for the same method for `TransformationMatrix`.